### PR TITLE
Docker alpine: add additional GDAL driver

### PR DESCRIPTION
**Pull request by mmacata**
In `alpine:3.18` many GDAL drivers were removed from the base package to reduce package size [(see related alpine aports issue here)](https://gitlab.alpinelinux.org/alpine/aports/-/issues/14910). Instead they can be installed individually. A list of all available additional drivers can be found [here](https://pkgs.alpinelinux.org/packages?name=gdal-driver*&branch=v3.18&repo=&arch=x86_64&maintainer=). Few were already removed from 3.15 to 3.17. 

This PR suggests to add the possibly most common ones.
Following a list of which drivers were removed from `gdal-tools` package between `alpine:3.17` and `alpine 3.18` :
```
vector
  4   BAG -raster,multidimensional raster,vector- (rw+v): Bathymetry Attributed Grid
  5   CAD -raster,vector- (rovs): AutoCAD Driver
  8   Carto -vector- (rw+): Carto
 15   Elasticsearch -vector- (rw+): Elastic Search
 16   FITS -raster,vector- (rw+): Flexible Image Transport System
 19   GMLAS -vector- (rwv): Geography Markup Language (GML) driven by application schemas
 32   JP2OpenJPEG -raster,vector- (rwv): JPEG-2000 driver based on OpenJPEG library
 34   LIBKML -vector- (rw+v): Keyhole Markup Language (LIBKML)
 37   MSSQLSpatial -vector- (rw+): Microsoft SQL Server Spatial Database
 42   MySQL -vector- (rw+): MySQL
 46   ODBC -vector- (ro):
 50   OGR_OGDI -vector- (ro): OGDI Vectors (VPF, VMAP, DCW)
 56   PCIDSK -raster,vector- (rw+v): PCIDSK Database File
 57   PDF -raster,vector- (rw+vs): Geospatial PDF
 61   PLSCENES -raster,vector- (ro): Planet Labs Scenes API
 62   PostgreSQL -vector- (rw+): PostgreSQL/PostGIS
 72   VFK -vector- (ro): Czech Cadastral Exchange Data Format
 76   XLS -vector- (ro): MS Excel format
 78   netCDF -raster,multidimensional raster,vector- (rw+s): Network Common Data Format

raster
 87   BAG -raster,multidimensional raster,vector- (rw+v): Bathymetry Attributed Grid
 94   CAD -raster,vector- (rovs): AutoCAD Driver
119   EXR -raster- (rw+vs): Extended Dynamic Range Image File Format
122   FITS -raster,vector- (rw+): Flexible Image Transport System
136   HDF5 -raster,multidimensional raster- (rovs): Hierarchical Data Format Release 5
137   HDF5Image -raster- (rov): HDF5 Dataset
138   HEIF -raster- (rov): ISO/IEC 23008-12:2017 High Efficiency Image File Format
150   JP2OpenJPEG -raster,vector- (rwv): JPEG-2000 driver based on OpenJPEG library
152   KEA -raster- (rw+v): KEA Image Format (.kea)
177   PCIDSK -raster,vector- (rw+v): PCIDSK Database File
178   PCRaster -raster- (rw+): PCRaster Raster File
179   PDF -raster,vector- (rw+vs): Geospatial PDF
183   PLSCENES -raster,vector- (ro): Planet Labs Scenes API
184   PNG -raster- (rwv): Portable Network Graphics
187   PostGISRaster -raster- (rws): PostGIS Raster driver
218   WEBP -raster- (rwv): WEBP
219   WMS -raster- (rwvs): OGC Web Map Service
225   netCDF -raster,multidimensional raster,vector- (rw+s): Network Common Data Format
```

What do you think about the selection I made @wenzeslaus @pesekon2 @neteler ?
Anyones redundant or missing?

### CODE REVIEW

**Comment by mmacata** [comment]
Difference in image size for stable-alpine as example doesn't seem to be much:
```
grass-gis         stable-alpine-additional-gdal-drivers   93b67e3165fb   28 seconds ago   652MB
osgeo/grass-gis   stable-alpine                           b10393ab033f   13 days ago      643MB
```

**Comment by pesekon2** [comment]
The selection makes sense to me. I could consider also `LIBKML` though, if it doesn't make a big difference in size.

**Comment by mmacata** [comment]
> The selection makes sense to me. I could consider also `LIBKML` or `GMLAS` though, if they don't make a big difference in size.

Only ~2MB. I added them both.
```
grass-gis             stable-alpine-more-additional-gdal-drivers   968a45f814ba   About a minute ago   654MB
grass-gis             stable-alpine-additional-gdal-drivers        93b67e3165fb   24 hours ago         652MB
osgeo/grass-gis       stable-alpine                                b10393ab033f   2 weeks ago          643MB
```

**Review by ninsbl — APPROVED** [review]
Thanks for that. Looks good!
My comments are just suggestions and could be added by users later as well (for those who need that)...

File: docker/alpine/Dockerfile [hunk 1]
```diff
@@ -24,6 +24,14 @@ ENV GRASS_RUN_PACKAGES="\
...
       gdal \
       gdal-dev \
+      gdal-driver-GMLAS \
+      gdal-driver-HDF5 \
+      gdal-driver-JP2OpenJPEG \
+      gdal-driver-LIBKML \
+      gdal-driver-netCDF \
+      gdal-driver-PG \
+      gdal-driver-PNG \
+      gdal-driver-WMS \
```
[the remark this answers is not in the document]
Comment (nilason) [reply docker/alpine/Dockerfile:34]: > I have not tested how much storage that occupies though...

Should be ~0.2 MB.

[the remark this answers is not in the document]
Comment (mmacata) [reply docker/alpine/Dockerfile:36]: I added all three suggested drivers as well, size can really be neglected.

**Review by neteler — APPROVED** [review]
Thanks!

**Pull request merged**